CMR/python/cmr/search/common.py

```diff
@@ -295,6 +314,8 @@ def apply_filters(filters, items):
                 result.append(filters(item))
     return result
 
+# document-it: {"key":"max-time", "default": "300000"}
+# document-it: {"from":"._make_search_request"}
 def search_by_page(base, query=None, filters=None, page_state=None, config:dict = None):
```

Comment:
same comment about style consistency (there's more of these so I'm thinking maybe you wanted it to be var_name:type instead of var_name: type and that the latter was the inconsistent one all along)